README.md

```diff
@@ -31,17 +31,12 @@ Read the proposal from https://github.com/moby/moby/issues/32925
 
 BuildKit daemon can be built in two different versions: one that uses [containerd](https://github.com/containerd/containerd) for execution and distribution, and a standalone version that doesn't have other dependencies apart from [runc](https://github.com/opencontainers/runc). We are open for adding more backends. `buildd` is a CLI utility for serving the gRPC API. 
 
-```bash
-# buildd daemon (choose one)
-go build -o buildd-containerd -tags containerd ./cmd/buildd
-go build -o buildd-standalone -tags standalone ./cmd/buildd
+The following command installs `buildd` and `buildctl` to `/usr/local/bin`:
```

Comment:
The lines above about building 2 different versions need to be updated as well

Reply:
done

Reply:
Sorry, I didn't mean adding more info how to build the separate versions but removing that(line 32) or modifying it so that it mentions daemon with 2 different workers(and how they differ) instead of 2 different daemons.

Reply:
oh sorry :sweat_smile: 

updated